org.mozilla.Firefox.json.in

```diff
@@ -0,0 +1,84 @@
+{
```

Comment:
I think we need both at this time.  As far as I can tell, generate-json is a developer tool that is run manually when preparing for an upgrade, and not run at build time.